packages/dag4-network/src/api/v2/block-explorer-api.ts

```diff
@@ -59,7 +75,7 @@ export class BlockExplorerV2Api {
     }
   }
 
-  _getTransactionSearchPathAndParams (basePath: string, limit, searchAfter, sentOnly, receivedOnly, searchBefore) {
+  _getTransactionSearchPathAndParams (basePath: string, limit: number = 0, searchAfter = '', sentOnly = false, receivedOnly = false, searchBefore = '') {
```

Comment:
I think `sentOnly` and `receivedOnly` belong in the calling function and can be included as part of `basePath`. Refactoring to that would clean up the order of the input params in this function.